v2_changes/terminology.adoc

```diff
@@ -1,4 +1,4 @@
-= Terminology
+= Terminology Changes
```

Comment:
Not sure about this one either, and I might be inclined to just leave it as Terminology.